### DISCUSSION TITLE
发包产出 CSS 文件，解绑 Less

### DISCUSSION THREAD

**Discussion by wwyx778** [alibaba/formily#2243] [Q&A 2.x] [answered]
现在 Formily 强关联 Less，但并不是所有人都会使用 Less 的。
Formily 能否在发布时打包产出 CSS 文件供用户引入，而不用再强制使用 Less？

**Comment by janryWang** [comment, marked as answer]
一直就有css，dist目录

**Reply by wwyx778** [reply]
但是没有用 less-loader 的情况下引入组件的时候还是会去 @formily/antd/esm 中去找 style.less 导致报错。这种情况应该怎么处理呢？

**Reply by janryWang** [reply]
你用了import-plugin吧

**Reply by wwyx778** [reply]
目前使用 babel-plugin-import 解决了这个问题。
